app/services/flood_risk_engine/simple_encoding.rb

```diff
@@ -0,0 +1,26 @@
+module FloodRiskEngine
+  # Simple cipher encoding from
+  # http://stackoverflow.com/questions/4128939/simple-encryption-in-ruby-without-external-gems
+  module SimpleEncoding
+
+    protected
+
+    def encode(text)
+      return "" unless text.present?
+      text.tr(characters, encoding)
+    end
+
+    def decode(text)
+      return "" unless text.present?
+      text.tr(encoding, characters)
+    end
+
+    def characters
+      "ABCDEFGHIJKLMNOPQRSTUVWXYZabcdefghijklmnopqrstuvwxyz0123456789"
+    end
+
+    def encoding
+      "N8nGfS1rpcHvmoiP6WB9FMQwXjyZ0Rhu73stKAla2Yd4LTJqkeUzgEOxbCD5VI"
+    end
+  end
+end
```

Comment:
Yes - that looks like a better solution. I looked at setting cookies to be encrypted by default, but the setting I was using interferred with the operation of the database session store, so went for the basic encoding option. Oh and can't use SECRET_KEY_BASE as just a cipher - so needs to include all the characters.